jest.config.js

```diff
@@ -40,7 +40,7 @@ module.exports = {
     '<rootDir>/test',
   ],
   moduleNameMapper: {
-    '\\.(scss)$': '<rootDir>/node_modules/identity-obj-proxy',
+    '\\.(scss|css)$': '<rootDir>/node_modules/identity-obj-proxy',
```

Comment:
Do we need this? I didn’t see css files (only scss).

Reply:
without css

```
Jest encountered an unexpected token

    This usually means that you are trying to import a file which Jest cannot parse, e.g. it's not plain JavaScript.

    By default, if Jest sees a Babel config, it will use that to transform your files, ignoring "node_modules".

    Here's what you can do:
     • To have some of your "node_modules" files transformed, you can specify a custom "transformIgnorePatterns" in your config.
     • If you need a custom transformation specify a "transform" option in your config.
     • If you simply want to mock your non-JS modules (e.g. binary assets) you can stub them out with the "moduleNameMapper" config option.

    You'll find more details and examples of these config options in the docs:
    https://jestjs.io/docs/en/configuration.html

    Details:

    /Users/ayushsharma/github/verdaccio/node_modules/normalize.css/normalize.css:11
    html {
         ^

    SyntaxError: Unexpected token {
```

It uses this 

https://github.com/verdaccio/verdaccio/blob/4112587d7298fb03cd5d5fd044a5de41d209b8e8/src/webui/app.js#L11

Reply:
A slightly shorter form is `\\.s?css$'`